engine/audio/private/fmod_debug.cpp

```diff
@@ -4,7 +4,14 @@
 #include "fmod_errors.h"
 #include "log.hpp"
 #include <iostream>
-#include <string>
+void CHECKRESULT_fn(FMOD_RESULT result, MAYBE_UNUSED const char* file, int line)
```

Comment:
Forgot to include common.hpp for the MAYBE_UNUSED macro. Also should rename to FMODCheckResult since other libraries can also have handling for their result types